eng/common/TestResources/New-TestResources.ps1

```diff
@@ -509,13 +532,15 @@ the SecureString to plaintext by another means.
 .EXAMPLE
 New-TestResources.ps1 `
     -BaseName 'Generated' `
-    -ServiceDirectory '$(ServiceDirectory)' `
-    -TenantId '$(TenantId)' `
-    -ProvisionerApplicationId '$(ProvisionerId)' `
-    -ProvisionerApplicationSecret '$(ProvisionerSecret)' `
-    -TestApplicationId '$(TestAppId)' `
-    -TestApplicationSecret '$(TestAppSecret)' `
+    -ServiceDirectory $(ServiceDirectory) `
+    -Location '$(Location)' `
     -DeleteAfterHours 24 `
+    -AdditionalParameters @{ additionalParam1 = 'value'; additionalParam2 = 'value' } `
```

Comment:
We should also think about whether or not we can do something similar for AdditionalParameters and maybe move TestApplication to additional parameters as well. Don't let that slow down your current work but just an idea we should think more about.

Reply:
Let's meet and discuss this. Having these isn't required to be passed, but was initially done to support CIs. There's no reason to remove it, though. We still need it for splatting.